src/app/components/cds/buttons-checkboxes-chips/Chip.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IconButton from '@material-ui/core/IconButton';
+import styles from './Chip.module.css';
+import CancelFillIcon from '../../../icons/cancel_fill.svg';
+
+const Chip = ({
+  label,
+  onRemove,
+  removable,
+}) => {
+  const handleRemove = () => {
+    onRemove();
+  };
+
+  return (
+    <div
+      className={`${styles['chip-container']} typography-body2`}
+    >
+      <span>{label}</span>
+      { removable && (
+        <IconButton
+          onClick={handleRemove}
+          className={styles['delete-button']}
+        >
+          <CancelFillIcon />
+        </IconButton>
+      )}
+    </div>
+  );
+};
+
+Chip.defaultProps = {
+  removable: false,
+  onRemove: () => {},
+};
+
+Chip.propTypes = {
+  onRemove: PropTypes.func,
+  removable: PropTypes.bool,
```

Comment:
This seems a bit redundant. Can we get rid of the removable prop and consider removable if onRemove is passed?

Reply:
makes sense, I'll change that

Reply:
fixed in [d8dc2cf](https://github.com/meedan/check-web/pull/1481/commits/d8dc2cf87946b8dc67eca4da7ee0ce09e93fbe28)